tasks/e2e-kitchensink.sh

```diff
@@ -153,11 +163,11 @@ npm link $root_path/packages/eslint-config-react-app
 npm link $root_path/packages/react-dev-utils
 npm link $root_path/packages/react-scripts
 
-# ...and we need  to remove template's .babelrc
+# ...and we need to remove template's .babelrc
 rm .babelrc
 
 # Test the build
-NODE_PATH=src REACT_APP_SHELL_ENV_MESSAGE=fromtheshell npm run build
+NODE_PATH=src NODE_ENV=production REACT_APP_SHELL_ENV_MESSAGE=fromtheshell npm run build
```

Comment:
Also went ahead and removed them from the `start` commands. We should only need it for the mocha calls.